wasm/derive/Cargo.toml

```diff
@@ -9,5 +9,7 @@ edition = "2021"
 proc-macro = true
 
 [dependencies]
-syn = {version = "1", default-features = false, features = ["full", "extra-traits"] }
+syn = {version = "1", default-features = false, features = ["full", "extra-traits", "derive"] }
 quote = "1.0"
+proc-macro2 = "1.0.50"
+convert_case = "0.6.0"
```

Comment:
Unfortunately I see we have both `heck` and `convert_case` as transitive dependencies so I guess either one is fine

Reply:
Actually, it should be possible to fix that.

Reply:
Haven't heard about `heck`. Actually I like `convert_case` more, but there is no difference here. Fixed